ui/scripts/accounts.js

```diff
@@ -1476,6 +1476,14 @@
                                                 form: {
                                                     title: 'label.action.change.password',
                                                     fields: {
+                                                        oldPassword: {
+                                                            label: 'label.old.password',
+                                                            isPassword: true,
+                                                            validation: {
+                                                                required: !(isAdmin() || isDomainAdmin())
```

Comment:
As we are hiding this field for admin and domain admin. I think we can assign the value true for the required parameter.